CMakeLists.txt

```diff
@@ -160,11 +168,15 @@ target_include_directories(bitsandbytes PUBLIC csrc include)
 
 if(BUILD_CUDA)
     target_include_directories(bitsandbytes PUBLIC ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
-    target_link_libraries(bitsandbytes PUBLIC CUDA::cudart CUDA::cublas CUDA::cusparse)
+    if(NOT USE_CUDA_WRAPPER)
+        target_link_libraries(bitsandbytes PUBLIC CUDA::cublas CUDA::cusparse)
+    endif()
     if(NO_CUBLASLT)
         target_compile_definitions(bitsandbytes PUBLIC NO_CUBLASLT)
     else()
-        target_link_libraries(bitsandbytes PUBLIC CUDA::cublasLt)
+        if(NOT USE_CUDA_WRAPPER)
+            target_link_libraries(bitsandbytes PUBLIC CUDA::cublasLt)
+        endif()
```

Comment:
CMake has `CMAKE_CUDA_RUNTIME_LIBRARY` to control this; maybe would be better here.

Reply:
https://cmake.org/cmake/help/latest/variable/CMAKE_CUDA_RUNTIME_LIBRARY.html
docuement says `CMAKE_CUDA_RUNTIME_LIBRARY` only control `cudart`
(Currently, with or without this PR, `cudart_static` added by default)